src/style.css

```diff
@@ -1 +1,119 @@
-/* add styles here */
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+  font-style: normal;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.search-form--big {
+  width: 100%;
+  height: 70px;
+}
+
+.search-form--small {
+  width: 100%;
+  height: 42px;
+}
+
+.search-form__label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.search-form__label::after {
+  position: absolute;
+  content: '';
+  display: block;
+}
+
+.search-form__label--big::after {
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.search-form__label--small::after {
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.search-form__input {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  font-family: inherit;
+  font-weight: 300;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  outline: none;
+}
+
+.search-form__input:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+}
+
+.search-form__input__big {
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search-form__input__small {
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.search-form__input::placeholder {
+  color: inherit;
+  font-weight: 300;
+}
+
+.search-form__input__big::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search-form__input__small::placeholder {
```

Comment:
The class name 'search-form__input__small::placeholder' should use a single underscore for BEM naming convention, like 'search-form__input--small::placeholder', to match the convention used elsewhere in the file.